src/relay/transforms/to_mixed_precision.cc

```diff
@@ -36,6 +36,7 @@
 namespace tvm {
 namespace relay {
 
+TVM_REGISTER_PASS_CONFIG_OPTION("relay.ToMixedPrecision.enable_original_type", Bool);
```

Comment:
This name is a bit confusing. Maybe something like "keep_orig_output_dtype".

Reply:
Changed the name to keep_orig_output_dtype